AMW_web/src/main/java/ch/puzzle/itc/mobiliar/presentation/copyResource/CopyResourceDataProvider.java

```diff
@@ -178,6 +182,7 @@ else if (copyFromResourceId == null) {
 			}
 			else {
 				GlobalMessageAppender.addSuccessMessage("Copy successful");
+				return true;
 			}
 		}
 		return false;
```

Comment:
Better remove `return true` at the end and move it into the if/else: compiler catches missing returns, else if doesn't have to be checked, the else if and else is not needed anymore:
 ```
		if (resourceToOverwrite == null) {
 			String message = "No resource selected.";
 			GlobalMessageAppender.addErrorMessage(message);
 			return false;
 		}
 	        if (copyFromResourceId == null) {
 			String message = "No resource to copy from selected.";
 			GlobalMessageAppender.addErrorMessage(message);
                        return false;
 		}
// remove else {
```